include/HUDL/HUDL.hpp

```diff
@@ -248,6 +270,22 @@ class HUDL {
             .Data = CO_LINK(0x2100, 3, 32),
         },
 
+        /**
+         * RPDO2 mapping, determines the PDO messages to send when RPDO1 is triggered
```

Comment:
```suggestion
         * RPDO2 mapping, determines the PDO messages to send when RPDO2 is triggered
```

Reply:
Should also be "message to _recive_" instead of send.